gwpy/time/_tconvert.py

```diff
@@ -263,19 +263,28 @@ def _str_to_datetime(datestr):
     # use maya
     try:
         import maya
-        return maya.when(datestr).datetime()
     except ImportError:
-        pass
+        MAYA_ERROR = None
```

Comment:
should MAYA_ERROR be none on ImportError? Is None how this error is encoded?

Reply:
This is eventually raise as a `RuntimeError` on L281-287. We hide the `ImportError` because we use `dateutil.parser` as a backstop if `maya` isn't available, so the `ImportError` isn't very informative for the user.